R/clear.cache.R

```diff
@@ -65,3 +65,8 @@ clear.cache <- function (...){
         }
 }
 
+# Determine if a variable is cached or not
+.is.cached <- function (variable) {
+        file.exists(file.path(.cache.dir, paste0(variable, ".RData")))
```

Comment:
Ah now I see why you mentioned the `if` statement. I think it would be nice to either make `.read.cache.info` able to check more than one variable at a time, or restore the `.is.cached` function. I want to read cached state as well in an upcoming PR (to add a `list.data` function), and I need to loop over all variables just to get the cached state. `file.exists` works fine with a character vector, but the part where the `cache.hash` is loaded doesn't work for more than one variable. I think `.is.cached` is useful standalone without loading the hashes, as well as being more descriptive than `.read.cache.info(variable)$in.cache`, and so I'd prefer `.read.cache.info` calling `.is.cached`. I'm sorry that I didn't express the point about a wider use of `.is.cached` more clearly before.